pkg/azureutils/azure_disk_utils.go

```diff
@@ -106,14 +106,20 @@ const (
 	TrueValue                 = "true"
 
 	// CRDs specific constants
-	PartitionLabel    = "azdrivernodes.disk.csi.azure.com/partition"
-	AzVolumeFinalizer = "disk.csi.azure.com/azvolume-finalizer"
+	PartitionLabel = "azdrivernodes.disk.csi.azure.com/partition"
+	// 1. AzVolumeAttachmentFinalizer for AzVolumeAttachment objects handles deletion of AzVolumeAttachment CRIs
+	// 2. AzVolumeAttachmentFinalizer for AzVolume prevents AzVolume CRI from being deleted before all AzVolumeAttachments attached to that volume is deleted as well
```

Comment:
typo : AzVolumeFinalizer?

Reply:
Ah, AzVolumeAttachmentFinalizer is also added to AzVolume to prevent its CRI deletion while there are AzVolumeAttachment CRIs referring to it, which is added and removed by AzVolumeAttachment controller.